tasker/validation/host_validator.py

```
@@ -26,22 +26,23 @@ class HostValidator:
     """
     
     @staticmethod
-    def validate_hosts(tasks, global_vars, task_results, exec_type=None, default_exec_type='pbrun', check_connectivity=False, debug_callback=None, log_callback=None):
+    def validate_hosts(tasks, global_vars, task_results, exec_type=None, default_exec_type='pbrun', check_connectivity=False, skip_command_validation=False, debug_callback=None, log_callback=None):
```

Ruff (0.14.2)

Unused static method argument: `check_connectivity`

(ARG004)